src/components/radio/index.tsx

```diff
@@ -16,23 +16,25 @@ interface Props {
   helperText?: any;
   options: Option[];
   isReadOnly?: boolean;
+  columnDirection?: boolean;
   onChange?: (event: ChangeEvent<any>) => void;
 }
 
-const Radio = ({
+const Radio: FC<Props> = ({
   name,
   value,
   error,
   labelText,
   helperText,
   options,
   isReadOnly,
+  columnDirection,
   onChange,
   ...props
-}: Props): JSX.Element => (
+}) => (
   <SC.Radio {...props}>
     {labelText && <SC.Label htmlFor={name}>{labelText}</SC.Label>}
-    <SC.Options>
+    <SC.Options columnDirection={columnDirection}>
```

Comment:
Her kan du bruke "transient props" siden columnDirection ikke er et attributt på span. Bare add dollar sign foran prop-name. Tror du får en feil på dette i browser-console?

If you want to prevent props meant to be consumed by styled components from being passed to the underlying React node or rendered to the DOM element, you can prefix the prop name with a dollar sign ($), turning it into a transient prop.